### PR TITLE
Optimize DIV for some common divizors

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

**Comment by rphmeier** [comment]
how were these divisors selected? benchmark results?

File: ethcore/src/evm/interpreter/mod.rs [hunk 1]
```diff
@@ -599,7 +610,19 @@ impl<Cost: CostType> Interpreter<Cost> {
...
+						TWO => a >> 1,
+						TWO_POW_5 => a >> 5,
+						TWO_POW_8 => a >> 8,
+						TWO_POW_16 => a >> 16,
+						TWO_POW_24 => a >> 24,
+						TWO_POW_64 => a >> 64,
+						TWO_POW_96 => a >> 96,
+						TWO_POW_224 => a >> 224,
+						TWO_POW_248 => a >> 248,
+						_ => a.overflowing_div(b).0,
```
Comment (rphmeier) [inline ethcore/src/evm/interpreter/mod.rs]: couldn't these be put into the implementation of `div` itself for `U256`?

Comment (tomusdrw) [inline ethcore/src/evm/interpreter/mod.rs]: I think it's better here - it's for evm's specific use cases. Uint is supposed to be general purpose library I don't think it suits well there. Probably would be better to use a different division algorithm there in additon.

**Comment by arkpar** [comment]
Selected by building a histogram for a few thousand recent blocks. This improves import time for some of the heavy blocks by a factor of two

**Pull request merged**